### PR TITLE
MB-3237 Add unit tests for parsers.py

### PR DESCRIPTION
**Pull request by sandy-wright**
## Description

This PR establishes the pattern for adding tests to this project, and adds initial tests for the `APIParser` class in `utils/parsers.py`.

## Setup

To run the tests, you will need to run through the setup instructions for the project that involve installing `pyenv` and creating a virtual environment. You can read these instructions here: https://github.com/transcom/milmove_load_testing#setup-pyenv

After that process has completed and you have activated your virtual environment, you may run the tests with:

```sh
pytest -v
```

## Code Review Verification Steps

* [x] Request review from a member of a different team.
* [x] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-3237) for this change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -256,6 +257,29 @@ If you encounter compiler issues while installing the required Python version, t
 brew unlink binutils
 ```
 
+### Testing
```
Comment (shimonacarvalho) [inline README.md:260]: Thanks for keeping the docs updated!

**Review by shimonacarvalho — APPROVED** [review]
LGTM - all tests pass.

**Comment by NamibiaLT** [comment]
The command `pytest -v` did not work for me, but when I wrote `python3 -m  pytest -v` I was able to run the tests and see that they are all passing.

**Comment by sandy-wright** [comment]
> The command `pytest -v` did not work for me, but when I wrote `python3 -m pytest -v` I was able to run the tests and see that they are all passing.

Thanks for noting this! I think this actually was the same issue Donald was running into - I think if you have an existing install of `pytest` that isn't in the virtual env, you need to use this command. I'll make a note to update the README to include this!

**Pull request merged**